src/modules/flow/calamari/calamari.c

```diff
@@ -274,7 +275,7 @@ calamari_7seg_new_type(const struct sol_flow_node_type **current)
     };
 
     nodes[SEG_CTL].type = SOL_FLOW_NODE_TYPE_CALAMARI_SEGMENTS_CTL;
-    nodes[SEG_CLEAR].type = nodes[SEG_LATCH].type = nodes[SEG_CLOCK].type = nodes[SEG_DATA].type = SOL_FLOW_NODE_TYPE_GPIO_WRITER;
+    nodes[SEG_CLEAR].type = nodes[SEG_LATCH].type = nodes[SEG_CLOCK].type = nodes[SEG_DATA].type = *(const struct sol_flow_node_type **)sol_symbol_get("flow", "gpio", SOL_FLOW_NODE_TYPE_GPIO_WRITER);
```

Comment:
this is still not good  and will crash if gpio is a module and is missing at runtime :-(